docs/advanced.md

```diff
@@ -162,3 +162,61 @@ client = httpx.Client(proxies=proxy)
 # This request will be tunneled instead of forwarded.
 client.get("http://example.com")
 ```
+
+
+## Timeout fine-tuning
+HTTPX library offers many request timeout management options. HTTPX distinguish three types of timeouts: **connect** timeouts, 
+**write** timeouts and **read** timeouts.
+
+* Connect timeout specify time period within which a connection between a client and a server must be established.
+* Write timeout specify the maximum duration before timing out writes of the response.
```

Comment:
Actually, the write timeout applies between the sending of individual *chunks* of *request* data:

```suggestion
* Write timeout specifies the maximum duration to wait for a chunk of data to be sent (for example, a chunk of the request body). If HTTPX is unable to send data within this time frame, a `WriteTimeout` exception is raised.
```

Hope I'm not mistaken that *writing* refer to *sending a request*. Makes me think that connect/sending/receive might be more intuitive wording perhaps?